Prox/Prox/Data/PlacesController.swift

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import AFNetworking
+import Foundation
+import Deferred
+
+/*
+ * Delegate methods for updating places asynchronously.
+ * All methods on the delegate will be called on the main thread.
+ */
+protocol PlacesControllerDelegate: class {
+    func placesController(_ controller: PlacesController, didUpdatePlaces places: [Place])
+    func placesControllerNoPlacesFound(_ controller: PlacesController, willRetry retry: Bool)
+    func placesController(_ controller: PlacesController, didError error: Error)
+}
+
+let apiSuffix = "/api/v1.0/at/%f/%f"
```

Comment:
as this is not used anywhere outside of `PlacesController` this should be marked as `private`